base/essentials.jl

```diff
@@ -102,7 +102,7 @@ g (generic function with 1 method)
 
 julia> @code_typed f([1.0])
 CodeInfo(
-1 ─ %1 = invoke Main.g(_2::AbstractArray)::Float64
+1 ─ %1 =    invoke g(A::AbstractArray)::Float64
```

Comment:
This indentation is strange. I wonder if that change was intentional. Might be worh bisecting it.